src/gatsby-node.js

```diff
@@ -1,9 +1,34 @@
 const simpleGit = require(`simple-git`)
 import { pluginOptionsSchema } from "./options-validation"
+import { getDate } from "./repo"
```

Comment:
Where is ./repo tho

Reply:
Whoops, left that in from testing. Nice catch.